components/MyEventsList/index.js

```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components";
+
+const StyledList = styled.li`
+  list-style: none;
+  margin: 30px;
+`;
+
+const Wrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  max-width: 1300px;
+`;
+
+const Paragraph = styled.p`
+  margin: 10px;
+`;
+
+const EventCard = styled.li`
+  box-shadow: 0px 0px 10px rgba(0, 0, 0, 0.3);
+  background-color: lightpink;
+  margin-top: 30px;
+  border-radius: 10px;
+`;
+
+const Title = styled.h2`
+  font-weight: 600;
+`;
+
+export default function MyEventsList({ ownEvents }) {
+  return (
+    <>
```

Comment:
You do not need the fragments here as you only render one JSX element -> StyledList